src/components/addSourceWizard/hardcodedComponents/aws/provisioningArn.js

```diff
@@ -0,0 +1,226 @@
+import React, { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+
+import {
+  Alert,
+  ClipboardCopy,
+  ClipboardCopyVariant,
+  Text,
+  TextContent,
+  TextList,
+  TextListItem,
+  TextListVariants,
+  TextVariants,
+} from '@patternfly/react-core';
+
+import { getSourcesApi } from '../../../../api/entities';
+
+const ROLE_NAME = 'RH-HCC-provisioning-role';
+const POLICY_COMMAND_STRING = `
+POLICY_ARN=$(aws iam create-policy --policy-name RH-HCC-provisioning-policy --policy-document '{
+"Version": "2012-10-17",
+"Statement": [
+  {
+    "Sid": "RedHatProvisioning",
+    "Effect": "Allow",
+    "Action": [
+      "iam:GetPolicyVersion",
+      "iam:GetPolicy",
+      "iam:ListAttachedRolePolicies",
+      "iam:GetRolePolicy",
+      "ec2:CreateKeyPair",
+      "ec2:CreateLaunchTemplate",
+      "ec2:CreateLaunchTemplateVersion",
+      "ec2:CreateTags",
+      "ec2:DeleteKeyPair",
+      "ec2:DeleteTags",
+      "ec2:DescribeAvailabilityZones",
+      "ec2:DescribeImages",
+      "ec2:DescribeInstanceTypes",
+      "ec2:DescribeInstances",
+      "ec2:DescribeKeyPairs",
+      "ec2:DescribeLaunchTemplates",
+      "ec2:DescribeLaunchTemplateVersions",
+      "ec2:DescribeRegions",
+      "ec2:DescribeSecurityGroups",
+      "ec2:DescribeSnapshotAttribute",
+      "ec2:DescribeTags",
+      "ec2:ImportKeyPair",
+      "ec2:RunInstances",
+      "ec2:StartInstances"
+    ],
+    "Resource": "*"
+  }
+]
+}' | jq '.Policy.Arn')`;
+
+const ROLE_COMMAND_STRING = `
+ROLE_ARN=$(aws iam create-role --role-name ${ROLE_NAME} --assume-role-policy-document '{
+  "Version": "2012-10-17",
+  "Statement": {
+    "Effect": "Allow",
+    "Principal": {
+      "AWS": "<PROVISIONING_AWS_ACCOUNT>"
+    },
+    "Action": "sts:AssumeRole"
+  }
+}' | jq '.Role.Arn')`;
+
+const ATTACH_COMMAND_STRING = `aws iam attach-role-policy --role-name ${ROLE_NAME} --policy-arn $POLICY_ARN`;
+
+export const IAMPolicyDescription = () => {
+  const intl = useIntl();
+
+  return (
+    <TextContent>
+      <Text component={TextVariants.p}>
+        {intl.formatMessage({
+          id: 'provisioning.iam.grantPermissions',
+          defaultMessage:
+            'To grant Red Hat permissions to RunInstances on your Amazon Web Services (AWS) Elastic Cloud (EC2), create following AWS Identity and Access Management (IAM) policy.',
```

Comment:
```suggestion
            'To grant Red Hat permissions to run instances on your Amazon Web Services (AWS) Elastic Cloud (EC2), create the following AWS Identity and Access Management (IAM) policy.',
```
Create the following AWS Identity and Access Management (IAM) policy to grant Red Hat permissions to run instances on your Amazon Web Services (AWS) Elastic Cloud (EC2).